### DISCUSSION TITLE
Elevation Map

### DISCUSSION THREAD

**Discussion by torresweb** [gpx-animator/gpx-animator#527] [Ideas]
Hi,

Possible an old idea, but I could not find in the roadmap... any plans to include an elevation map?

--
Pedro

**Comment by McPringle** [comment]
Yes, there are two elevation-related issues in our backlog. Issue #373 is related to your question. Another one is #184. I can't say when I will be able to implement this feature, I'm very busy at the moment. If somebody else is providing a PR, I'll highly appreciate it.
